client/my-sites/plan-features/item.jsx

```diff
@@ -27,15 +28,25 @@ export default function PlanFeaturesItem( {
 		onMouseLeave( event.currentTarget, description );
 	};
 
+	const mouseEvents = {
+		onMouseEnter: handleOnMouseEvent,
+		onMouseLeave: handleOnMouseLeave
```

Comment:
Oh, just noticed that `handleOnMouseEvent` is not part of this PR. Alrighty then. :)